libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1813,7 +1813,7 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         }
     }
 
-    private fun createChildScope() = CoroutineScope(threadController.getMainScopeAndRootJob().job)
+    private fun createChildScope() = CoroutineScopeFactory.createScope(threadController.getMainScopeAndRootJob().job)
```

Comment:
maybe?
```suggestion
    private fun createChildScope() = threadController.getMainScopeAndRootJob().scope
```

Reply:
I thought we wanted to create a new scope here.

Reply:
`getMainScopeAndRootJob` creates a new scope, doesn't it?

Reply:
Right, makes sense.